### PR TITLE
E2E fluentd build fix

### PR DESCRIPTION
**Pull request by petergvizd**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?

Fix of e2e fluentd image build

### Why?

Failed e2e-test in CircleCI

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Comment by petergvizd** [comment]
I am really sorry @ahma , but I forgot to mark my previous PR  https://github.com/banzaicloud/logging-operator/pull/989 as draft and some WIP commits ended in master, so this is fix to use locally build fluentd image in E2E tests in CircleCI.

**Review by ahma — APPROVED** [review]
@petergvizd np, thanks for the quick fix

File: fluentd-image/e2e-test/healthy.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -x
+
+# Liveness probe is aimed to help in situarions where fluentd
```
Comment (aslafy-z) [inline fluentd-image/e2e-test/healthy.sh:3]:
```suggestion
# Liveness probe is aimed to help in situations where fluentd
```

**Review by aslafy-z — COMMENTED** [review]
I spotted a typo, see comment

**Pull request merged**